<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseWidget</class>
 <widget class="QDialog" name="BaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>416</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>416</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>416</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizon_titlebar"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizon_center"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizon_statusbar"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
